File: src/main/java/com/dersaun/apigestaocontas/domain/dtos/ResumoFormaPagamentoMensalDTO.java
package com.dersaun.apigestaocontas.domain.dtos;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ResumoFormaPagamentoMensalDTO {

    private String titulo;

    private BigDecimal valor;

    private Double porcentagem;

    private String corIcone;
}

File: src/app/conta/dtos/resumo-forma-pagamento-mensal.dto.ts
export class ResumoFormaPagamentoMensalDTO {
  titulo: string;
  valor: number;
  porcentagem: number;
  corIcone: string;

  constructor(dto?: Partial<ResumoFormaPagamentoMensalDTO>) {
    if (dto) {
      Object.assign(this, dto);
    }
  }
}
